src/compiler/parser/lines_1000_2000.rs

```diff
@@ -61,15 +61,15 @@ impl ParserType {
             self.parse_statement()
         });
         Debug_.assert(self.token() == SyntaxKind::EndOfFileToken, None);
-        let end_of_file_token = self.add_jsdoc_comment(self.parse_token_node());
+        let end_of_file_token = self.add_jsdoc_comment(self.parse_token_node().into());
```

Comment:
`.add_jsdoc_comment()` ended up needing a wrapped `Rc<Node>` so that "infected" several parser method return types